packages/@okta/vuepress-site/docs/reference/api/devices/index.md

```diff
@@ -0,0 +1,2323 @@
+* Device profile attibutes should follow formats specified in [device profile properties](#profile-object).
+
+>**Note** `PUT` can't be used for partial updates. Unspecified attribute values will be deleted.
+
+##### Request Path Parameters
+
+| Parameter   | Type   | Description                                       |
+| ----------- | ------ | ------------------------------------------------- |
+| `deviceId`  | String | The `id` of device object                         |
+
+##### Request Query Parameters
+
+None.
+
+##### Request Body
+
+Update [request](#update-request)
+
+##### Response Body
+
+Updated [Device](#device-model)
+
+##### Example Usage
+
+###### Request
+
+>**Note** This request would update the `status` to `SUSPENDED`, along with changes in the device `profile` properties. If you wish to simply update device `profile`, do not include `status` in the request.
+
+```bash
+curl -v -X PUT \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "status" : "SUSPENDED",
+    "profile": {
+        "displayName": "John Device",
+        "platform" : "MACOS",
+        "manufacturer": "Apple Inc",
+        "model": "Macbook Pro 15"
+    }
+}' "https://{yourOktaDomain}/api/v1/devices/guo4a5u7YAHhjXrMK0g4"
+```
+
+###### Response
+
+```json
+{
+    "id": "guo4a5u7YAHhjXrMK0g4",
+    "status": "SUSPENDED",
+    "created": "2019-10-02T18:03:07.000Z",
+    "lastUpdated": "2019-10-07T17:20:37.000Z",
+    "profile": {
+        "displayName": "John Device",
+        "platform": "MACOS",
+        "manufacturer": "Apple Inc",
+        "model": "Macbook Pro 15",
+        "osVersion": null,
+        "serialNumber": null,
+        "imei": null,
+        "meid": null,
+        "udid": null,
+        "sid": null
+    },
+    "_links": {
+        "unsuspend": {
+            "href": "https://{yourOktaDomain}/api/v1/devices/guo4a5u7YAHhjXrMK0g4/lifecycle/unsuspend",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        },
+        "self": {
+            "href": "https://{yourOktaDomain}/api/v1/devices/guo4a5u7YAHhjXrMK0g4",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PATCH",
+                    "PUT"
+                ]
+            }
+        },
+        "users": {
+            "href": "https://{yourOktaDomain}/api/v1/devices/guo4a5u7YAHhjXrMK0g4/users",
+            "hints": {
+                "allow": [
+                    "GET"
+                ]
+            }
+        },
+        "deactivate": {
+            "href": "https://{yourOktaDomain}/api/v1/devices/guo4a5u7YAHhjXrMK0g4/lifecycle/deactivate",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        }
+    }
+}
+```
+
+### Update Device with PATCH
+
+<ApiOperation method="patch" url="/api/v1/devices/${deviceId}" />
```

Comment:
#765 adds support for this. We'll want to rebase from `master` once that PR is merged.